src/xrpld/app/misc/CredentialHelpers.cpp

```diff
@@ -225,6 +226,50 @@ makeSorted(STArray const& in)
     return out;
 }
 
+NotTEC
```

Comment:
This is good. Can I bother you to make a drive-by fix, replacing the non-printable character in line 169 in this file ?

```
"Credential doesn<E2><80><99>t belong to the source account. Cred: "
```

Reply:
Wow! Where this fancy apostrophe  came from? Maybe I copy/pasted it from GH? Fixed.